### PR TITLE
 libmaa: update to 1.4.2. 

### PR DESCRIPTION
**Pull request by Chocimier**

### CODE REVIEW

File: srcpkgs/dictd/template [hunk 1]
```diff
@@ -1,17 +1,18 @@
-# Template file for 'dictd', based on the ArchLinux package by Sergej Pupykin
+# Template file for 'dictd'
+# based on the ArchLinux package by Sergej Pupykin
```
Comment (maxice8) [inline srcpkgs/dictd/template]: this can be removed

File: srcpkgs/dictd/template [hunk 2]
```diff
@@ -1,17 +1,18 @@
...
+revision=5
 build_style=gnu-configure
-configure_args="--enable-dictorg --sbindir=/usr/bin --sysconfdir=/etc/dict"
+configure_args="--enable-dictorg --sysconfdir=/etc/dict"
 conf_files="/etc/dict/dictd.conf /etc/dict/site.info"
 hostmakedepends="flex libtool"
 makedepends="zlib-devel libmaa-devel"
 short_desc="DICT protocol server"
 maintainer="Christopher Brannon <chris@the-brannons.com>"
 license="GPL-2"
```
Comment (maxice8) [inline srcpkgs/dictd/template]: GPL-2.0-or-later

File: srcpkgs/libmaa/template [hunk 3]
```diff
@@ -1,15 +1,25 @@
...
+version=1.4.2
 revision=1
-build_style=gnu-configure
-hostmakedepends="flex libtool"
+build_style=gnu-makefile
+make_cmd=mkcmake
+hostmakedepends="flex libtool mk-configure"
 short_desc="Provides many low-level data structures useful for writing compilers"
 maintainer="Christopher Brannon <chris@the-brannons.com>"
 license="GPL-2"
```
Comment (maxice8) [inline srcpkgs/libmaa/template]: GPL-2.0-or-later

**Comment by Chocimier** [comment]
Changed.

**Pull request merged**